cmd/hubproxy/main.go

```diff
@@ -19,70 +18,47 @@ import (
 	"log/slog"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 	"tailscale.com/tsnet"
 )
 
-var (
-	configFile string
-	cfg        config.Config
-)
+var configFile string
 
 func newRootCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "proxy",
 		Short: "HubProxy - A robust GitHub webhook proxy",
-		Long: `HubProxy is a robust webhook proxy to enhance the reliability and security 
-of GitHub webhook integrations. It acts as an intermediary between GitHub 
+		Long: `HubProxy is a robust webhook proxy to enhance the reliability and security
+of GitHub webhook integrations. It acts as an intermediary between GitHub
 and your target services.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			viper.AutomaticEnv()
+			viper.SetEnvPrefix("hubproxy")
+			viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
```

Comment:
Re-above: It's instead yo can make ENV_VARS be `_` style. I wonder if you can do the same for configs...